### ISSUE TITLE
custom tflite model run failed with Gpudelegate  (android P)

### ISSUE THREAD

**Issue by sunzhe09** [tensorflow/tensorflow#28657]
<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:bug_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow):
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
- TensorFlow installed from (source or binary):pip
- TensorFlow version (use command below):1.12.2
- Python version:3.5
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version:
- GPU model and memory:

You can collect some of this information using our environment capture
[script](https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh)
You can also obtain the TensorFlow version with: 1. TF 1.0: `python -c "import
tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"` 2. TF 2.0: `python -c
"import tensorflow as tf; print(tf.version.GIT_VERSION, tf.version.VERSION)"`

**Describe the current behavior**
build GPUdelegate failed on android P   java demo
**Describe the expected behavior**
build pass
**Code to reproduce the issue**
Provide a reproducible test case that is the bare minimum necessary to generate the problem.
I convert my model like this:
freeze_graph --input_graph=eval.pb --input_checkpoint=./model_quant_self/model.ckpt-19-19 --output_graph=frozen_eval_graph.pb --output_node_names=Softmax
tflite_convert --output_file=poolnet_gzq.tflite --graph_def_file=./model_gzq.pb --inference_type=FLOAT --input_arrays=Placeholder --input_shapes=1,224,224,3 --output_arrays=oup

**Other info / logs**
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

here is the log:

2019-05-13 16:25:04.870 20456-22983/? W/System.err: java.lang.RuntimeException: java.lang.IllegalArgumentException: Internal error: Failed to apply delegate: GpuDelegate Prepare: fuse_auto_input failedNode number 133 (GpuDelegate) failed to prepare.

2019-05-13 16:25:04.870 20456-22983/? W/System.err: Caused by: java.lang.IllegalArgumentException: Internal error: Failed to apply delegate: GpuDelegate Prepare: fuse_auto_input failedNode number 133 (GpuDelegate) failed to prepare.
failedNode133 is my outputNode Sigmiod/Softmax ,I had tried ,both are failed

**Comment by impjdi** [comment]
@sunzhe09 

Is it possible for you to share the .tflite file with me?  If weights are confidential, you can share an uninitialized graph with random weights; all I care about is the network structure.

**Comment by sunzhe09** [comment]
@impjdi  My model is from here :https://github.com/zhangludl/A-bi-directional-message-passing-model-for-salient-object-detection,you can download the model from the line Download the pretrained model from here, and put it under the ./model directory. If you transform it to tflite,you can reproduce my issue. I am sorry I don't have right to upload my file ...

**Comment by mariloubylr** [comment]
In case it helps; I had a similar problem and was able to reproduce the error with the following simple keras model:

`input = keras.Input(shape=(68,128,34))

left = keras.layers.Conv2D(22, (3,3), padding="same", activation="relu")(input2)
right = keras.layers.Conv2D(22, (1,1), padding="same", activation="relu")(input2)

center = keras.layers.add([left, right])

left = keras.layers.add([right, center])
right = keras.layers.Conv2D(22, (3,1), padding="same", activation="relu")(center)

end = keras.layers.add([left,right])


model = keras.Model(input2, end)

sess = keras.backend.get_session()

converter = lite.TFLiteConverter.from_session(sess, [model.input], [model.output])
model = converter.convert()
file = open( 'reproduce.tflite' , 'wb' ) 
file.write( model )
`
